CHANGELOG.md

```diff
@@ -58,6 +58,7 @@ Ref: https://keepachangelog.com/en/1.0.0/
 
 ### Improvements
 
+* (rosetta) [#13832](https://github.com/cosmos/cosmos-sdk/pull/13832) Correctly populates rosetta's `/network/status` endpoint response. Roseta data api is divided into its own go files (account, block, mempool, network).
```

Comment:
Can you move that here: https://github.com/cosmos/cosmos-sdk/blob/main/tools/rosetta/CHANGELOG.md?